doc/classes/PhysicsDirectBodyState.xml

```diff
@@ -154,7 +154,7 @@
 	</methods>
 	<members>
 		<member name="angular_velocity" type="Vector3" setter="set_angular_velocity" getter="get_angular_velocity">
-			The body's rotational velocity.
+			The body's rotational velocity in [i]degrees[/i] per second..
```

Comment:
```suggestion
			The body's rotational velocity in axis-angle format. The magnitude of the vector is the rotation rate in [i]degrees[/i] per second.
```

Reply:
Bump.

Reply:
@aaronfranke Does this only apply to PhysicsDirectBodyState, or also to RigidBody and RigidBody2D?

Reply:
I just checked the code and I can confirm it's the same format, using radians per second in both cases.